RptToXml/ConditionFormulas.cs

```diff
@@ -69,7 +69,10 @@ private CRReportDefModel.PrintOptions GetRASRDMPrintOptionsObject(string name, R
 
 			return _rcd.ReportDocument.PrintOptions;
 		}
-
+		private CRDataDefModel.ISCRField GetRASField(FormulaFieldDefinition ff)

```

Comment:
Shouldn't this be named GetDisplayNameFormulaField?

Reply:
Well all the methods above access the _rcd  object and are called GetRAS.... . The name ends with the return type.
For example:
private CRReportDefModel.PrintOptions GetRASRDMPrintOptionsObject(string name, ReportDocument report)
private GroupAreaFormat GetRASDDMGroupAreaFormatObject(Group group, ReportDocument report)
private CRReportDefModel.ISCRReportObject GetRASRDMReportObject(string oname, ReportDocument report)

My method access the _rcd member varable
  I thought a similar naming schema would make sense.

All this is in a region "#region Get ReportAppServer Objects"

Reply:
I see, so GetRASDisplayNameFormulaField then? It's not getting some arbitrary field, it's getting crFieldDisplayNameFormula right?

Reply:
I had a second look at it.
It can be a bit more generic.
It returns an ISCRField from a FieldDefinition. TO do so it uses the name of the FieldDefinition to find the corresponding ISCRField.
How about:
`private CRDataDefModel.ISCRField GetRASCRFieldByName(FieldDefinition ff)
		{
			return _rcd.DataDefinition.FormulaFields.FindField(ff.FormulaName, CRDataDefModel.CrFieldDisplayNameTypeEnum.crFieldDisplayNameFormula);
		}`

Reply:
Perfect